src/Reports/SiteSummary.php

```diff
@@ -67,21 +54,61 @@ public function columns()
     public function getReportField()
     {
         Requirements::css('silverstripe-maintenance/css/sitesummary.css');
-
-        /** @var GridField $gridField */
-        $gridField = parent::getReportField();
```

Comment:
Can you please add this doc block back in?

Reply:
👍 Bad merge conflict resolution I guess :\ - Adding a docblock for the `$config` var right below it too.

Reply:
That is less necessary, but won't hurt